config/definitions.yml

```diff
@@ -40,23 +40,10 @@ sle12_common: &sle12_common
     - "http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP2/SLE_12_SP2/"
   :iso_url: "http://dist.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/Products:/CASP10/images/iso/SUSE-CaaS-Platform-1.0-DVD-x86_64-Build*-Media1.iso"
 
-:storage-ng:
-  :dud_dist: "tw"
-  :yast_url: "http://download.opensuse.org/repositories/YaST:/storage-ng/openSUSE_Tumbleweed/"
-  :iso_url: "http://download.opensuse.org/repositories/YaST:/storage-ng/images/iso/StorageNG-Test-Build*-Media.iso"
-
-:sles15:
-  :dud_dist: "sle15"
-  :yast_url: "http://download.suse.de/ibs/Devel:/YaST:/Head/SUSE_SLE-15_GA/"
-  :iso_url: "http://download.suse.de/ibs/SUSE:/SLE-15:/GA/images/iso/SLE-15-Server-DVD-x86_64-Build*-Media1.iso"
-  :extra_repos:
-    - :server: "http://download.suse.de/ibs/SUSE:/SLE-15:/GA/standard/"
-      :packages: ["lsof", "perl-Config-IniFiles", "libcamgm100", "SuSEfirewall2", "vsftpd" ]
-
 :leanos15:
```

Comment:
We are replacing sles15 key by leanos and at the end it is the only reference to leanos 😢